CITATION.cff

```diff
@@ -0,0 +1,138 @@
+abstract: >
+    MetPy is an open-source and community-driven Python package for meteorology designed to fit well within the scientific Python stack (numpy, matplotlib, etc.).
+    Its goal is to bring the scripted weather analysis capabilities of GEMPAK (and tools like it) to the powerful scientific Python ecosystem.
+    The guiding principle is to make MetPy easy to use with any dataset that can be read into Python.
+    MetPy's general functionality breaks down into reading data, meteorological calculations, and meteorology-specific plotting.
+authors: 
+  - family-names: May
+    given-names: Ryan
+    affiliation: "UCAR/Unidata"
+    orcid: "https://orcid.org/0000-0003-2907-038X"
+  - family-names: Arms
+    given-names: Sean
+    affiliation: "UCAR/Unidata"
+    orcid: "https://orcid.org/0000-0001-9835-113X"
+  - family-names: Marsh
+    given-names: Patrick
+  - family-names: Bruning
+    given-names: Eric
+    affiliation: "Texas Tech University"
+    orcid: "https://orcid.org/0000-0003-1959-442X"
+  - family-names: Leeman
+    given-names: John
+    orcid: "https://orcid.org/0000-0002-3624-1821"
+  - family-names: Goebbert
+    given-names: Kevin
+    affiliation: "Valparaiso University"
+    orcid: "https://orcid.org/0000-0001-7559-2432"
+  - family-names: Thielen
+    given-names: Jonathan
+    orcid: "https://orcid.org/0000-0002-5479-0189"
+  - family-names: Bruick
+    given-names: Zachary
+    orcid: "https://orcid.org/0000-0002-0299-9845"
+  - family-names: Camron
+    given-names: "M. Drew"
+    affiliation: "UCAR/Unidata"
+    orcid: "https://orcid.org/0000-0001-7246-6502"
+cff-version: "1.1.0"
+contact:
+  - name: Unidata
+    city: Boulder
+    region: Colorado
+    country: US
+    email: "support@unidata.ucar.edu"
+    tel: "303-497-8643"
+    fax: "303-497-8690"
+    website: "https://www.unidata.ucar.edu"
+date-released: 2021-07-30
```

Comment:
Did you want to update the date?
```suggestion
date-released: 2021-08-06
```

Reply:
Yeah, I was just waiting until we knew for sure when we were releasing, heh.